saleor/graphql/checkout/mutations.py

```diff
@@ -734,7 +734,10 @@ def perform_mutation(cls, _root, info, checkout_id, store_source, **data):
 
         discounts = info.context.discounts
         user = info.context.user
-        clean_checkout(checkout, list(checkout), discounts)
+
+        clean_args = checkout, list(checkout), discounts, CheckoutErrorCode
```

Comment:
This looks strange, I think it would be cleaner if we explicitly passed these arguments.

Reply:
Do we have to pass `list(checkout)` as an argument? It could be done inside the function and we would have one less argument.